common/src/lib.rs

```diff
@@ -8,6 +8,8 @@ pub type Intensity = u16;
 pub type FrameNumber = u32;
 pub type SampleRate = u64;
 
+use rdkafka::config::ClientConfig;
```

Comment:
Could this be moved up so that it is between the end of modules and start of types, this is the general format used elsewhere in the codebase.